### PR TITLE
Decrease amount of node saves

### PR DESCRIPTION
**Pull request by vuntz**
Saving is not cheap, and most of the time, it's not needed (as no attribute changed, or because we will already save later).

### CODE REVIEW

**Comment by nicolasbock** [comment]
There seems to be another race condition during the pacemaker setup. Or something is failing :(

File: chef/cookbooks/crowbar-pacemaker/recipes/mutual_ssh.rb [hunk 1]
```diff
@@ -22,27 +22,26 @@
...
+cluster_access_keys = []
 
 CrowbarPacemakerHelper.cluster_nodes(node).each do |cluster_node|
   pkey = nil
   if cluster_node[:crowbar][:ssh]
     pkey = cluster_node[:crowbar][:ssh][:root_pub_key]
   end
-  if !pkey.nil? && cluster_node.name != node.name && !access_keys.values.include?(pkey)
-    access_keys[cluster_node.name] = pkey
+  if !pkey.nil? && cluster_node.name != node.name && !access_keys.include?(pkey)
```
Comment (Itxaka) [inline chef/cookbooks/crowbar-pacemaker/recipes/mutual_ssh.rb]: `access_keys` does not exists anymore, is now `cluster_access_keys`

Comment (Itxaka) [inline chef/cookbooks/crowbar-pacemaker/recipes/mutual_ssh.rb]: Maybe this is the issue that breaks the CI job `00:29:53.340 Cannot find a resource for access_keys on suse version 12.2`

**Comment by nicolasbock** [comment]
@Itxaka nice catch! That explains the gating failure.

**Comment by nicolasbock** [comment]
The tempest failure in `TestVolumeBootPatternV2` is failing in all HA builds right now. @aplanas and I are looking into it.

**Comment by vuntz** [comment]
Was completed in https://github.com/crowbar/crowbar-ha/pull/210

**Pull request closed without merging**